docs/kics-auto-remediation.md

```diff
@@ -0,0 +1,30 @@
+# KICS AUTO REMEDIATION
+
+With this new feature, KICS provides auto remediation for simple replacements and simple additions in a single line.
+
+Note that this feature will be only available for Terraform, for now.
+
+<p align="center">
+<img width="950" alt="image" src="https://user-images.githubusercontent.com/74001161/177953750-3d279868-8cdb-44c9-86f2-379b05bb85d4.png">
+</p>
+
+
+
+## HOW KICS AR WORKS?
+
+
+1. As a first step, you will need to scan your project/file and generate a JSON report.
+
+   Example: ```docker run -v /home/cosmicgirl/:/path/ kics scan -p /path/sample.tf -i "41a38329-d81b-4be4-aef4-55b2615d3282,a9dfec39-a740-4105-bbd6-721ba163c053,2bb13841-7575-439e-8e0a-cccd9ede2fa8" --no-progress -o /path/results --report-formats json```
```

Comment:
Do we really need to add the -i parameter?
It may cause confusion, since if the query doesn't have auto-remediation suggestion, the AR will be calculated anyway, right?

Reply:
No. It is only an example. I will delete it. Thank you